### PR TITLE
cypress test for documentation menu item

### PR DESCRIPTION
**Pull request by ShaiahWren**
Hi, Zita & Martin,
This pr tests the external link for the Documentation tab.
See https://issues.redhat.com/browse/AAH-629

Any q's feel free to contact me!

### CODE REVIEW

File: test/cypress/fixtures/example.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (himdel) [inline test/cypress/fixtures/example.json:1]: this file gets created by cypress init, we're not using it for anything so no need to add it :)

File: test/cypress/integration/menu.js [hunk 2]
```diff
@@ -1,60 +1,71 @@
 describe('Hub Menu Tests', () => {
-    let baseUrl = Cypress.config().baseUrl;
```
Comment (himdel) [inline test/cypress/integration/menu.js:2]: you may want to reconfigure your editor not to reformat the whole file, it hides the actual changes

**Review by himdel — COMMENTED** [review]
LGTM except for the reformatting

(though feel free to also unify the test/ formatting with the rest in a separate PR if that's preferred :))

File: test/cypress/fixtures/example.json (hunk 1 above)
Comment (ShaiahWren) [reply test/cypress/fixtures/example.json:1]: I will blow it away!

File: test/cypress/integration/menu.js (hunk 2 above)
Comment (ShaiahWren) [reply test/cypress/integration/menu.js:2]: Do you mean... get rid of line 2?

Comment (ShaiahWren) [reply test/cypress/integration/menu.js:2]: O I see, Don't get rid of line 2. 
What should I run so the editor doesn't reformat the whole file?

**Comment by ZitaNemeckova** [comment]
@ShaiahWren This needs rebase otherwise it's good to go. Let me know if you need help :)

**Review by ZitaNemeckova — APPROVED** [review]

**Pull request merged**